blueprints/app/files/_js_eslint.config.mjs

```diff
@@ -0,0 +1,119 @@
+import globals from 'globals';
+import js from '@eslint/js';
+
+import ember from 'eslint-plugin-ember';
+import emberRecommended from 'eslint-plugin-ember/configs/recommended';
+import gjsRecommended from 'eslint-plugin-ember/configs/recommended-gjs';
+
+import prettier from 'eslint-plugin-prettier/recommended';
+import qunit from 'eslint-plugin-qunit';
+import n from 'eslint-plugin-n';
+
+import emberParser from 'ember-eslint-parser';
+import babelParser from '@babel/eslint-parser';
+
+const esmParserOptions = {
+  ecmaFeatures: { modules: true },
+  ecmaVersion: 'latest',
+  requireConfigFile: false,
+  babelOptions: {
+    plugins: [
+      ['@babel/plugin-proposal-decorators', { decoratorsBeforeExport: true }],
+    ],
+  },
+};
+
+export default [
+  js.configs.recommended,
+  prettier,
+  {
+    ignores: ['dist/', 'node_modules/', 'coverage/', '!**/.*'],
+    linterOptions: {
```

Comment:
I tried this setup in a v5.12 project and I noticed the global ignores config wasn't working as expected.

It seems that [`ignores` _must_ be the only key in the object](https://eslint.org/docs/latest/use/configure/ignore#ignoring-files) to be considered the "global ignore" config.

We should split the `linterOptions` into a separate object.

Reply:
makes sense -- thank you for trying this out!